### PR TITLE
Migrate apps to the new runtime

### PR DESCRIPTION
**Pull request by alekseysavostin**

### CODE REVIEW

File: src/dev.ts [hunk 1]
```diff
@@ -29,47 +29,40 @@
 //
```
Comment (kalabukdima) [inline src/dev.ts:29]: Let's delete this too. No commented code should be put under vcs

File: src/dev.ts [hunk 2]
```diff
@@ -78,19 +71,25 @@ appHost.start({
...
+
+    db: "streamdb-01",
   },
-});
+};
+
+appHost.start(mangroveStrategiesApp);
+
+if (false) {
+  appHost.start(parseBlocksApp);
```
Comment (kalabukdima) [inline src/dev.ts:94]: You can just name it `_parseBlocksApp` if you are just trying to avoid the warning

**Review by kalabukdima — APPROVED** [review]

File: src/dev.ts (hunk 1 above)
Comment (ivanbenko) [reply src/dev.ts:29]: why? we use this kind of files for development/manual testing so it's handy to have parts of the code commented as well

**Review by ivanbenko — APPROVED** [review]

File: src/dev.ts (hunk 2 above)
Comment (alekseysavostin) [reply src/dev.ts:94]: Not working for me: "62:7  warning  '_parseBlocksApp' is assigned a value but never used  @typescript-eslint/no-unused-vars
"

**Pull request merged**